src/config/index.ts

```diff
@@ -13,4 +14,5 @@ export const publicClient = createPublicClient({
 export const getConfig = (): Config => ({
   ...getEnv(),
   ...getConstants(),
+  ...getCustomThemes(),
 });
```

Comment:
I think it's kinda weird to access an environment variable and a theme config from the same object.
What if we use a property for each type of thing?

```suggestion
export const getConfig = (): Config => ({
  env: ...getEnv(),
  constants: ...getConstants(),
  customTheme: ...getCustomThemes(),
});
```

Could use destructuring assignment to access each object:
```
const { customTheme } = getConfig();
const { getMui: getMuiThemeConfig } = customTheme;
```

Reply:
[051c63d](https://github.com/defi-wonderland/web3-react-boilerplate/pull/21/commits/051c63d0c3dce26ec23203f015deabf3b3adf64e)